### ISSUE TITLE
Unable to include css files from installed packages

### ISSUE THREAD

**Issue by emepyc** [thgh/rollup-plugin-scss#24]
I'm trying to combine some scss / css files into one bundle using this package. In my entry.js I import all the stylesheets with:

```
import "awesomplete/awesomplete.css";
import "./simple-modal-custom.css";
import "./custom.scss";
```

The first one if from an installed module (ie in `node_modules/awesomplete/awesomplete.css`.
But this files get ignored and only the other 2 get bundled.
If I use relative imports this seems to work:

```
import "../node_modules/awesomplete/awesomplete.css";
```

I would expect the first version to work. Am I missing something?

**Comment by thgh** [comment]
That should be equivalent to `import "awesomplete/awesomplete.css";`

**Comment by emepyc** [comment]
yes, it should be the same (I have updated the OP with the correct import), but `import "awesomplete/awesomplete.css"` is not bundling the awesomplete css file for me

**Comment by thgh** [comment]
Do you mind creating a test repository?

This plugin has no default excludes. (rollup-plugin-sass excludes node_modules/** by default)

**Comment by frederikhors** [comment]
Me too:

`@import "bootstrap/scss/bootstrap.scss";` doesn't work,

`@import "../node_modules/bootstrap/scss/bootstrap.scss";` works.

**Comment by jonathanmarvens** [comment]
Any updates on this? __CC:__ @thgh

**Comment by thgh** [comment]
I don't know how node-sass resolves imports, if anyone could look into resolving node_modules, that would solve this?

**Comment by aharris** [comment]
You should be able to include node_modules:

```
    scss({
      includePaths: [
        'node_modules/'
      ]
    }),
```

**Comment by thgh** [comment]
@aharris Cool, I released version 2.0.0 with that includePath by default.
